README.md

```diff
@@ -3,3 +3,20 @@ libadiff - Audio diffing library
 
 [![Build Status](https://travis-ci.org/concert/libadiff.svg?branch=master)](
     https://travis-ci.org/concert/libadiff)
+
+## Build:
+If you're on a Debian-based Linux system, prerequisites are:
```

Comment:
Might be better to say what the dependencies are, then maybe the debian packages rather than making anyone non-deb work backwards.

Reply:
I agree, and defer to your cross-distribution expertise to name the packages correctly. I just wrote down what I did so that I didn't forget ;-)

Reply:
I think it would be something like "glib2 and libsndfile with headers"